docs/faq/code-analysis/does-codacy-place-limits-on-the-code-analysis.md

```diff
@@ -76,3 +76,4 @@ If you believe that you may have hit any of these limits and need help to overco
 ## See also
 
 -   [Which metrics does Codacy calculate?](which-metrics-does-codacy-calculate.md)
+-   [Why is my file over 150 KB missing?](../troubleshooting/why-is-my-file-over-150-kb-missing.md)
```

Comment:
This link is already in the table above, under Rationale, so maybe we don't need it in See Also. Otherwise, we should also add the other two links under Rationale to the See Also, for consistency.

Reply:
Good point! That must have been the reason why I hadn't originally included the link in the "See also" section. :smile: